src/components/ui/ConfirmationBox.js

```diff
@@ -0,0 +1,108 @@
+// @flow
+
+import React from 'react'
+
+import Button from './Button'
+
+import theme from '../../styles/theme'
+import { typeface } from '../../styles/typo'
+
+const cmz = require('cmz')
+
+type Props = {
+  title?: string,
+  content?: string,
+  action?: () => void,
+  actionLabel?: string,
+  dismissAction?: () => void
+}
+
+const WRAPPER_PADDING = '24px'
+const cx = {
+  wrapper: cmz(`
+    display: flex
+    align-items: center
+    flex-direction: column
+    font-size: 14px
+    justify-content: center
+    padding: ${WRAPPER_PADDING}
+    width: calc(500px - ${WRAPPER_PADDING})
+  `),
+
+  title: cmz(
+    typeface.semiHeading, `
+    color: ${theme.baseDark}
+    font-size: 24px
+    font-weight: 500
+    margin: 0
+    width: 100%
+  `),
+
+  content: cmz(
+    typeface.text,
+    `
+    color: ${theme.baseTuna}
```

Comment:
Please use appropriate `typo` color, check instructions on https://deploy-preview-467--xpui-develop.netlify.com/?path=/info/styleguide-general--colorpalette

in this case the correct should be:

```suggestion
    color: ${theme.typoParagraph}
```